build/Dockerfile

```diff
@@ -1,12 +1,7 @@
-FROM centos:7
+FROM fedora:30
```

Comment:
Commenting for the commit message: Please specify what was the problem (both in the body and if possible in the title as well). Please say why it was wrong in its original state.

s/it's/its/
s/an/and/

Reply:
Done